### ISSUE TITLE
Refine obj import to blender, color attribute not found

### ISSUE THREAD

**Issue by thomaswan** [pengHTYX/Era3D#30]
Import it3000-mc256.obj to blender, color attribute can be used as material base color. However, import refine__name__.obj file to blender, color attribute is not found.

**Comment by altava-sgp** [comment]
Usually blender can't get vertex color from obj file. ( In my case blender 3.6.2 )
You can export as ply with vertex color from original obj file by tool like [MeshLab](https://www.meshlab.net/#download).
There is also issue about this. https://github.com/pengHTYX/Era3D/issues/14#issuecomment-2151641469

**Comment by bananaman1983** [comment]
it's a clipping problem after the training. simple clipping like the code below may solve the problem where values sticking out above 1.0f

> def optim_clr(case, img_path, mesh_dir, save_dir, device):
>     ...             
>     np_colors= to_numpy(color_model.colors)         
>     np.clip(np_colors, 0.0, 1.0, out=np_colors) #clamp    
>     tensor_colors = torch.tensor(np_colors ,dtype=torch.float32,device=device)
>     
>     save_obj(vert, face, f'{save_dir}/refine_{case}.obj', tensor_colors.detach())  
>     ...

**Comment by altava-sgp** [comment]
@bananaman1983 Could you guess about this issue too ? https://github.com/pengHTYX/Era3D/issues/26

**Comment by bananaman1983** [comment]
dunno TBH. would like to take a look into though. it's weird

**Comment by miaowu99** [comment]
@bananaman1983 Excellent! you also solved the issue of noise in the colors of my exported mesh.
by:
```
  texture_color = torch.clamp(color_model.colors.detach(), min=0, max=1)
  save_obj(vert, face, f'{save_dir}/refine_{case}.obj', texture_color)
```

**Comment by altava-sgp** [comment]
@miaowu99 Great Job !
You solved this issue ! https://github.com/pengHTYX/Era3D/issues/4 👍 

### before
![image](https://github.com/user-attachments/assets/0d271c3e-6d2a-4916-8a85-ab79f83b0123)

### after
![image](https://github.com/user-attachments/assets/34d1d6e6-580c-4648-b705-98e20b804313)

@pengHTYX You can check it too !
